apps/web/src/content/blog/2026-03-01.md

```
@@ -0,0 +1,82 @@
+---
+title: "こんにちは！"
+publishedAt: "2026-03-01"
+tags: ["ポエム"]
+draft: false
+description: "新サイトを作った!"
+---
+
+## あいさつ
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)